webknossos/test.sh

```diff
@@ -12,6 +12,7 @@ if [ $# -eq 1 ] && [ "$1" = "--refresh-snapshots" ]; then
         mkdir -p binaryData
         export DOCKER_TAG=master__16177
         docker-compose pull webknossos
+        # TODO: either remove pg/db before starting or run tools/postgres/apply_evolutions.sh
```

Comment:
todo?

Reply:
Sorry, this is just something for the future, probably fine for now within this PR.

Reply:
Is there an issue for this TODO? 🙈

Reply:
Good point, I added an issue/epic for this and many more problems^^
https://github.com/scalableminds/webknossos-libs/issues/514